drivers/wifi/nxp/Kconfig.nxp

```diff
@@ -0,0 +1,1253 @@
+# Copyright 2022-2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig WIFI_NXP
+	bool "NXP Wi-Fi driver support"
+	select NET_L2_WIFI_MGMT if NETWORKING
+	select WIFI_USE_NATIVE_NETWORKING if NETWORKING
+	select NET_L2_ETHERNET_MGMT if NETWORKING && NET_L2_ETHERNET
+	select SDHC if !SOC_SERIES_RW6XX
+	select SDIO_STACK if !SOC_SERIES_RW6XX
+	select WIFI_NM if WIFI_NM_WPA_SUPPLICANT
+	depends on DT_HAS_NXP_WIFI_ENABLED
+	help
+	  Enable NXP SoC Wi-Fi support.
+
+if WIFI_NXP
+
+module = WIFI_NXP
+
+config WIFI_MGMT_SCAN_CHAN_MAX_MANUAL
+	default 50
+
+config NET_DHCPV4_SERVER_ADDR_COUNT
+	default 32
+
+config NET_TCP_WORKQ_STACK_SIZE
+	default 2048
+
+config NET_RX_STACK_SIZE
+	default 2048
+
+config NET_MGMT_EVENT_STACK_SIZE
+	default 2048
+
+config SYSTEM_WORKQUEUE_STACK_SIZE
+	default 2048
+
+config SHELL_STACK_SIZE
+	default 5120
+
+config POSIX_MAX_FDS
+	default 16
+
+config SHELL_ARGC_MAX
+	default 32
+
+config NET_BUF_DATA_SIZE
+	default 1744
+
+config NET_TC_TX_COUNT
+	default 1
+
+config NET_TC_RX_COUNT
+	default 1
+
+config NET_MGMT_EVENT_QUEUE_SIZE
+	default 20
+
+config NET_IF_MAX_IPV4_COUNT
+	default 2
+
+config NET_IF_MAX_IPV6_COUNT
+	default 2
```

Comment:
These do not really belong to driver as these are application specific configuration options